release-notes.md

```diff
@@ -9,6 +9,18 @@ The following updates were made in this release:
 * Added support for Griffon debug API.
 * Analytics now ignores the `global.ssl` configuration flag.
 
+## Aug 30, 2019
+
+The following updates were made in this release:
+
+**iOS Mobile Services 2.0.2**
+
+* Added the ability to trigger a In-App Messaging based off Analytics rules consequnse.
```

Comment:
Please reword so that it reads as follows:
"Added the ability to trigger an In-App message that is based on the Analytics rules consequence."